test/test_fx.py

```diff
@@ -1,4 +1,5 @@
 # Owner(s): ["module: fx"]
+# ruff: noqa: F841
```

Comment:
fyi @angelayi as fx owner, we most likely should fix this and the other fx files to make sure there is no hidden bug here